pkgs/tools/backup/duplicity/default.nix

```diff
@@ -76,6 +76,7 @@ let self = python3.pkgs.buildPythonApplication rec {
     cryptography
     ecdsa
     idna
+    invoke
```

Comment:
I am guessing this is from paramiko ssh backend so something like the following might be clearer:

```nix
] ++ paramiko.optional-dependencies.invoke ++ [
```